test/addurl_test.go

```diff
@@ -69,8 +69,10 @@ TAG ` + tag
 	})
 	defer os.RemoveAll(buildDir)
 
-	err := runRockerBuildWdWithOptions(buildDir, "-cache-dir", cacheDir)
-	assert.Nil(t, err, "no difference in downloaded and added files")
+	err := runRockerBuildWd(buildDir, "-cache-dir", cacheDir)
+	if err != nil {
```

Comment:
Why would you do that? It does not bring anything over a simple `!= nil` checking. And it will also output stuff to Stdout and will duplicate what `t.Fatal` will write anyway.
Also, once the thing is working functionally, it's the matter of convention. If there is no convention, then it's the matter of taste and integrity. Unit tests are written using the above style.